src/Neo/ProtocolSettings.cs

```diff
@@ -58,12 +58,16 @@ public record ProtocolSettings
         public string[] SeedList { get; init; }
 
         /// <summary>
-        /// Indicates the time in milliseconds between two blocks.
+        /// The milliseconds between two blocks in Neo.
+        /// For code that needs the block generation time based on blockchain state, 
+        /// use NeoSystemExtensions.GetBlockGenTime extension method instead.
         /// </summary>
         public uint MillisecondsPerBlock { get; init; }
```

Comment:
We can use obsolete to avoid the use, only in the new method

Reply:
But this value is actually not obsolete, it's still used until Echidna is enabled. And it's used to initialize Policy at Echidna height, so this value will never be removed from the node's configuration.